--- source_subfolder/tensorflow/lite/delegates/gpu/gl/egl_context_old.cc	2019-08-23 08:29:48.000000000 +0000
+++ source_subfolder/tensorflow/lite/delegates/gpu/gl/egl_context.cc	2019-08-23 08:30:44.000000000 +0000
@@ -15,6 +15,8 @@

 #include "tensorflow/lite/delegates/gpu/gl/egl_context.h"

+#include <cstring>
+
 #include "tensorflow/lite/delegates/gpu/common/status.h"
 #include "tensorflow/lite/delegates/gpu/gl/gl_call.h"
 #include "tensorflow/lite/delegates/gpu/gl/gl_errors.h"
@@ -54,7 +56,7 @@
 }

 bool HasExtension(EGLDisplay display, const char* name) {
-  return strstr(eglQueryString(display, EGL_EXTENSIONS), name);
+  return std::strstr(eglQueryString(display, EGL_EXTENSIONS), name);
 }

 }  // namespace
